trikot-kword/kword/src/commonMain/kotlin/com/mirego/trikot/kword/DebugI18N.kt

```diff
@@ -0,0 +1,31 @@
+package com.mirego.trikot.kword
+
+class DebugI18N : I18N {
+    override operator fun get(key: KWordKey): String {
+        return getTranslationKey(key)
+    }
+
+    override fun changeLocaleStrings(strings: Map<String, String>) {
+        // NO-OP
+    }
+
+    override fun changeLocaleStrings(source: KWordSource) {
+        // NO-OP
+    }
+
+    override fun t(key: KWordKey): String {
```

Comment:
nitpick: we could use the single expression notation in pretty much all these functions.